app/src/main/java/com/example/grassroots/utils/Constant.java

```diff
@@ -0,0 +1,9 @@
+package com.example.grassroots.utils;
+
+public class Constant {
+
+    public interface KEYS {
```

Comment:
Constant interfaces are poor uses of interfaces suggest looking at enums or just have them be public static final constants in the Constant class